src/commands/decrypt_passphrase.js

```diff
@@ -33,35 +33,37 @@ const passphraseOptionDescription = `Specifies a source for providing an encrypt
 		- --passphrase stdin (takes the first line only)
 `;
 
-const processInputs = (iv, passphrase) => ({ password, data }) =>
+const processInputs = encryptedPassphrase => ({ password, data }) =>
 	cryptography.decryptPassphrase({
-		cipher: passphrase || getFirstLineFromString(data),
-		iv,
+		encryptedPassphrase: encryptedPassphrase || getFirstLineFromString(data),
 		password,
 	});
 
-export const actionCreator = vorpal => async ({ iv, passphrase, options }) => {
+export const actionCreator = vorpal => async ({
+	encryptedPassphrase,
+	options,
+}) => {
 	const passphraseSource = options.passphrase;
 	const passwordSource = options.password;
 
-	if (!passphrase && !passphraseSource) {
+	if (!encryptedPassphrase && !passphraseSource) {
 		throw new ValidationError('No encrypted passphrase was provided.');
 	}
 
 	return getInputsFromSources(vorpal, {
 		password: {
 			source: passwordSource,
 		},
-		data: passphrase
+		data: encryptedPassphrase
 			? null
 			: {
 					source: passphraseSource,
 				},
-	}).then(processInputs(iv, passphrase));
+	}).then(processInputs(encryptedPassphrase));
 };
 
 const decryptPassphrase = createCommand({
-	command: 'decrypt passphrase <iv> [passphrase]',
+	command: 'decrypt passphrase [encryptedPassphrase]',
```

Comment:
does `encryptedPassphrase` needs to be optional?

Reply:
It's an optional argument because it can be passed via other means (e.g. stdin) as an option.